aiosu/v2/client.py

```diff
@@ -1761,37 +1761,47 @@ async def get_score(
         self,
         score_id: int,
         mode: Gamemode,
-        **kwargs: Any,
-    ) -> Union[Score, LazerScore]:
+    ) -> Score:
         r"""Gets data about a score.
 
         :param score_id: The ID of the score
         :type score_id: int
         :param mode: The gamemode to search for
         :type mode: aiosu.models.gamemode.Gamemode
-        :param \**kwargs:
-            See below
-
-        :Keyword Arguments:
-            * *new_format* (``bool``) --
-                Optional, whether to use the new score format, defaults to ``False``
 
         :raises APIException: Contains status code and error message
         :raises RefreshTokenExpiredError: If the client refresh token has expired
         :return: Score data object
         :rtype: aiosu.models.score.Score
         """
         url = f"{self.base_url}/api/v2/scores/{mode}/{score_id}"
-        headers = {}
-        new_format = kwargs.pop("new_format", False)
```

Comment:
the new_format logic should still remain in both methods to not break existing shit

this stuff should only be removed when the library fully switches to new api models